AnimalsTypes/Fox.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GameEngine.Entities;
+using GameEngine.Helpers;
+using Repository;
+
+namespace AnimalsTypes
+{
+    /// <summary>
+    /// Fox entity.
+    /// </summary>
+    public class Fox : Carnivore
+    {
+        /// <summary>
+        /// Fox entity.
+        /// </summary>
+        public Fox()
+        {
+            Id = GenerateId();
+            Sex = (AnimalSex)Helper.random.Next(2);
+            IsPaired = false;
+            Vision = ConstantsRepository.FoxVision;
+            Health = ConstantsRepository.MaxHealth;
```

Comment:
Same MaxHealth for different animals?

Reply:
For this task it was not described. I thouught that in this case Max health would be for all animals the same.